frontend/src/form/Form.types.ts

```diff
@@ -177,6 +177,20 @@ export interface ChartWidget extends Widget {
   chart: Chart;
 }
 
+export interface LabelWidget extends Widget {
+  label: string;
+  stringValue: string;
+  style: LabelWidgetStyle;
+}
+export interface LabelWidgetStyle {
```

Comment:
Please add and empty line between `LabelWidget` and `LabelWidgetStyle`

Reply:
done